api/app/controllers/api/v1/assignments_controller.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+module Api::V1
+  # Controller for Assignments
+	class AssignmentsController < ApplicationController
```

Comment:
Please fix this formatting.

Reply:
Do you mean formatting as in the tabbing?

Reply:
Yes !